frameworks/FSharp/oxpecker/src/App/Db.fs

```diff
@@ -95,15 +100,15 @@ module Db =
         for i in 0..results.Length-1 do
             let randomNumber = Random.Shared.Next(1, 10001)
             let struct(rnParamName, idParamName) = paramNames[i]
-            let random = NpgsqlParameter<int>(ParameterName = rnParamName, DbType = DbType.Int32, TypedValue = randomNumber)
-            command.Parameters.Add(random) |> ignore
-            let id = NpgsqlParameter<int>(ParameterName = idParamName, DbType = DbType.Int32, TypedValue = results[i].id)
-            command.Parameters.Add(id) |> ignore
+            command.Parameters.Add(NpgsqlParameter<int>(
+                ParameterName = rnParamName, DbType = DbType.Int32, TypedValue = randomNumber)) |> ignore
+            command.Parameters.Add(NpgsqlParameter<int>(
+                ParameterName = idParamName, DbType = DbType.Int32, TypedValue = results[i].id)) |> ignore
             results[i] <- { results[i] with randomnumber = randomNumber }
 
     let doMultipleUpdates (count: int) =
         task {
-            use conn = new NpgsqlConnection(ConnectionString)
+            use conn = new NpgsqlConnection(MultiplexedConnectionString)
```

Comment:
Did you find out that multiplexing was detrimental in Fortunes (we did too, for these machines at least)

Reply:
Yes, I did, you are right